style.css

```diff
@@ -0,0 +1,458 @@
+  justify-content: center;
+}
+
+
+/*
+section {
+display: flex;
+}
+
+.section__title {
+	color: #e51937;
+	font: 70px 'montserratsemibold';
+	position: absolute;
+}
+
+.article {
+width: 440px;
+	color: white;
+	background-color: #E51937;
+margin: 0;
+
+}
+
+.section__pic {
+float: left;
+}
+
+.article {
+	color: white;
+	background-color: #E51937;
+
+}
+
+.article__info {
+	margin: 20px 0 0 10px;
+	padding: 10px;
+
+}
+
+.article__charset {
+	margin: 0 0 10px;
+	display: flex; 	
+}
+
+.article__top {
+	margin: 20px 30px 40px 20px;
+}
+.article__title {
+	text-align: left;
+	padding: 10px;
+	font-size: 30px;
+}
+.article__text {
+	text-align: left;
+}
+
+.article__charset-item {
+	margin: 20px 10px;
+	
+}
+.charset {
+	display: flex;
+	align-items: center;
+	justify-content: center;	
+}
+.charset__item {
+	margin: 0 10px;
+}
+
+.charset__item-one {
+	display: flex;
+	align-items: baseline;
+	text-align: left;
+}
+
+.charset__item-two {
+    display: flex;
+	flex-direction: column;
+}
+.charset__one-big {
+	font: 35px 'montserratsemibold';
+	text-align: left;
+	
+}
+
+.charset__one-small {
+	font: 20px 'montserratsemibold';
+}
+
+.charset__button-out {
+	border: none;
+	color: white;
+display: flex;
+}
+
+
+.charset__button-pic {
+	margin-right: 5px;
+}
+.charset__button-border {
+border: 2px solid white;
+	color: white;
+}
+
+
+.button__text {
+	text-transform: uppercase;
+}
+
+
+
+
+
+footer {
+	display: flex;
+	color: white;
+}
+
+.footer__box {
+	background-color: #E51937;
+	width: 440px;
+}
+
+.footer__inform {
+	padding: 20px;
+}
+
+.footer__car {
+
+margin:;
+}
+
+.footer__model {
+	z-index: 2;	
+}
+
+.footer__footer {
+	flex-direction: column;
+	margin: 30px;
+}
+ 
+ .footer__title {
+ 	font: 30px 'montserratmedium';
+
+ }
+ .footer__text {
+	padding: 0 0 30px 0;
+}
+ 
+ 
+ .footer__info {
+ 
+	padding: 20px;
+ }
+ .footer__subtitle {
+ 	font: 30px 'montserratmedium';
+	padding: 20px;
+ }
+
+.footer__content {
+	font: 17px 'montserratbold';
+	
+}
+
+.footer__content-item {
+margin: 10px;
+}
+
+.content__items {
+margin: 0 40px;
+text-align: center;
+justify-content: center;
+display: flex;
+align-items: baseline;
+}
+
+.item {
+		align-items: center;
+	justify-content: center;
+	display: flex;
+}
+
+.footer__word-small {
+	font: 10px 'montserratregular';
+	opacity: 0.7;
+	
+}
+
+.footer__safe {
+	margin: 20px 0;
+	font: 50px 'montserratsemibold';
+}
+
+.safe {
+	align-items: center;
+	justify-content: center;
+	display: flex;
+}
+
+.safe__items {
+	margin: 40px 20px;
+}
+
+.safe__content-one {
+	align-items: center;
+	justify-content: center;
+	display: flex;
+}
+.safe__content-two {
+	font: 25px 'montserratmedium';
+}
+.footer__word {
+	margin-right: 10px;
+}
+
+.footer__small{
+	margin: 0;
+
+}
+.footer__button-out {
+	
+	border: none;
+}
+
+.footer__button-border {
+	color: white;
+}
+
+.footer__button-text {
+	text-transform: uppercase;
+	color: white;
+	font-size: 13px;
+}
+
+.footer__button-pic {
+	margin-right: 5px;
+}
+
+*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)